numpy/core/fromnumeric.py

```diff
@@ -3243,12 +3243,14 @@ def around(a, decimals=0, out=None):
     See Also
     --------
     ndarray.round : equivalent method
-
+    numpy.round_ : equivalent method
     ceil, fix, floor, rint, trunc
 
 
     Notes
     -----
+    np.round is often used as an alias for np.around.
```

Comment:
Using single backticks will make these terms link to their definitions.

```suggestion
    `round` is often used as an alias for `around`.
```

If you want to be very specific about the namespaces (which may make sense in this case) you can instead use the fully-qualified name, e.g. `` `numpy.round` ``.

If you *really* want to use the `np` alias (and have the terms not be links), then these should be wrapped in double-backticks so that they are formatted as code, e.g. ``` ``np.round`` ```